recipes/blat/meta.yaml

```diff
@@ -14,8 +14,6 @@ build:
 
 requirements:
   build:
-    - libpng
-    - zlib
   run:
```

Comment:
Is this recipe even needed? It's probably identical to `ucsc-blat`.

Reply:
Probably not, I will remove it.